mautic/mautic
# Add customMjml to API responses.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | https://github.com/mautic/developer-documentation-new/pull/211
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Set up the API Settings and API Credentials(Optional).
3. Send a request to Mautic to get an email which uses MJML templating.
4. The response data should have a new property `['grapesjsbuilder']['customMjml']` along with `customHtml` & all the other data of the said email.
